app/javascript/components/patient/Patient.js

```diff
@@ -163,85 +253,168 @@ class Patient extends React.Component {
                 </Row>
                 <Row>
                   <Col className="text-truncate">
-                    <span>
-                      {this.props.details.address_line_1 && `${this.props.details.address_line_1}`}
-                      {this.props.details.address_line_2 && ` ${this.props.details.address_line_2}`}
-                      {this.props.details.foreign_address_line_1 && `${this.props.details.foreign_address_line_1}`}
-                      {this.props.details.foreign_address_line_2 && ` ${this.props.details.foreign_address_line_2}`}
-                    </span>
-                    <br />
-                    <span>
-                      {this.props.details.address_city ? this.props.details.address_city : ''}
-                      {this.props.details.address_state ? ` ${this.props.details.address_state}` : ''}
-                      {this.props.details.address_county ? ` ${this.props.details.address_county}` : ''}
-                      {this.props.details.address_zip ? ` ${this.props.details.address_zip}` : ''}
-                      {this.props.details.foreign_address_city ? this.props.details.foreign_address_city : ''}
-                      {this.props.details.foreign_address_country ? ` ${this.props.details.foreign_address_country}` : ''}
-                      {this.props.details.foreign_address_zip ? ` ${this.props.details.foreign_address_zip}` : ''}
-                    </span>
-                    <br />
+                    <Row className="py-1">
```

Comment:
should we do a:

{this.state.showDomesticAddress && this.state.showForeignAddress && ...} for displaying the "HOME ADDRESS" title?

Reply:
done - although one of these will always be defined